packages/gatsby-docs/src/pages/run/running_the_tool.mdx

```diff
@@ -1,36 +1,67 @@
 ---
-title: Running the tool
+title: Install and run the tool
 ---
 
-Complete the following steps to configure and run the tool and to review and implement any required changes.
+The UI Upgrade Helper is available for customers from SPM Software Downloads on [Merative Support](https://merative.my.site.com/mysupport/s/).
 
-1. Run `docker login`.
+You must be a technical contact to download software, open a support case if you need access. Select **SPM Software Download** and enter your technical contact credentials to download the software. 
 
-2. Download the appropriate batch file or shell script for your environment:
+The UI Upgrade Helper is available as an internal download for Merative employees.
+
+Complete the following steps to install, configure, and run the tool and to review and implement any required changes.
+
+1. Download `spm-ui-upgrade-helper.tar`.
+2. Load the Docker images:
 
    Windows ™️
 
-   `curl -L https://raw.githubusercontent.com/IBM/spm-ui-upgrade-helper/main/spm-ui-upgrade-helper.bat -o spm-ui-upgrade-helper.bat`
+   `docker load <./spm-ui-upgrade-helper.tar`
 
    Linux ®️
    
+   `docker load <./spm-ui-upgrade-helper.tar`
+
+3. Open Docker Desktop and check for the following images.
+
+4. Download the appropriate batch file or shell script for your environment:
+
+   Windows ™️
+
+   `curl -L https://raw.githubusercontent.com/IBM/spm-ui-upgrade-helper/main/spm-ui-upgrade-helper.bat -o spm-ui-upgrade-helper.bat`
```

Comment:
we don't need to download spm-ui-upgrade-helper.bat. or spm-ui-upgrade-helper.sh we need just docker-compose.yml